### PR TITLE
[11.0] [MIG] contract show invoice

### PR DESCRIPTION
**Pull request by levkar**

### CODE REVIEW

File: contract_show_invoice/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (feketemihai) [inline contract_show_invoice/README.rst:1]: @levkar Please update readme, images with png, Odoo version in runbot..

**Comment by feketemihai** [comment]
@levkar Please squash transifex commits..

**Review by feketemihai — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
This module is for showing invoices not generated by contracts, but having the analytic account, so I would change the name of the module and also move the module to OCA/account-analytic.

**Comment by levkar** [comment]
I agree. I was wondering why it is in contracts :)

**Comment by pedrobaeza** [comment]
Closing then.

**Pull request closed without merging**